src/test/java/core/basesyntax/service/amount/SupplyHandlerTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.amount;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class SupplyHandlerTest {
+
+    private static final FruitTransaction bananaBalance;
+    private static final FruitTransaction bananaSupply;
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
+    private final OperationHandler balance = new BalanceHandler();
+    private final OperationHandler supply = new SupplyHandler();
+
+    static {
```

Comment:
instead of using static block of initialization use methods with @BeforeClass annotation, fix it in every test class